src/components/modal/ConfirmModalRoot/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC, useCallback } from 'react';
+import { useConfirmModalState } from '@/atoms/confirmModalState';
+import ModalPortal from '@/components/modal/ModalPortal';
+import ConfirmModal from '@/components/modal/ConfirmModal';
+
+const ConfirmModalRoot: FC = () => {
+  const [confirmModalState, setConfirmModalState] = useConfirmModalState();
+  const {
+    isOpen,
+    message,
+    cancelContent,
+    acceptContent,
+    cancelHandler,
+    acceptHandler,
+  } = confirmModalState;
+
+  const closeHandler = useCallback(() => {
+    setConfirmModalState((prevState) => ({ ...prevState, isOpen: false }));
+  }, []);
+
+  return (
+    <ModalPortal>
+      <ConfirmModal
+        isOpen={isOpen}
+        message={message}
+        cancelContent={cancelContent}
+        acceptContent={acceptContent}
+        cancelHandler={cancelHandler}
+        acceptHandler={acceptHandler}
+        closeHandler={closeHandler}
```

Comment:
`cancelHandler, acceptHandler` 는 각각 취소, 확인 시 요청하는 함수이고 
`closeHandler`는 모달을 닫는 함수로 예상됩니다.
하지만 모두 `Handler`라는 suffix로 묶여있어 비슷한 역할을 하는 함수로 인식되어집니다
혹시 closeModal 이라는 명칭을 사용하는건 어떨까요?

```suggestion
        cancelHandler={cancelHandler}
        acceptHandler={acceptHandler}
        closeModal={closeModal}
```

Reply:
맞습니다, `acceptHandler, cancelHandler`와는 성격이 살짝 다른 것 같네요. 

좋습니당

Reply:
수정했습니다!

관련 커밋: [367bd85](https://github.com/telbby/telbby-frontend/pull/25/commits/367bd8576883e0c717fb485e9a408c7f3391dae4)